### PR TITLE
Update pytest.auth_headers with auth access_token headers

### PR DESCRIPTION
**Pull request by krishnaa05**
Link to JIRA ticket if there is one: [GQE-358](https://ctds-planx.atlassian.net/browse/GQE-358)

### Deployment changes

[GQE-358]: https://ctds-planx.atlassian.net/browse/GQE-358?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: gen3-integration-tests/services/requestor.py [hunk 1]
```diff
@@ -116,9 +115,14 @@ def get_request_id(self, user: str):
...
-            f"{self.BASE_URL}/{request_id}", headers=pytest.auth_headers[user]
-        )
+        auth = Gen3Auth(refresh_token=pytest.api_keys[user], endpoint=self.BASE_URL)
+        access_token = auth.get_access_token()
+        headers = {
+            "Accept": "application/json",
+            "Authorization": f"bearer {access_token}",
+            "Content-Type": "application/json",
+        }
+        status_res = requests.get(f"{self.BASE_URL}/{request_id}", headers=headers)
```
Comment (haraprasadj) [inline gen3-integration-tests/services/requestor.py:125]: can we use auth curl here?

Comment (krishnaa05) [reply gen3-integration-tests/services/requestor.py:125]: We cant use curl here, as curl doesn't accept headers as input
https://uc-cdis.github.io/gen3sdk-python/_build/html/_modules/gen3/auth.html#Gen3Auth.curl

Comment (haraprasadj) [reply gen3-integration-tests/services/requestor.py:125]: why do we need the headers? it is a simple get call, no?

File: gen3-integration-tests/services/indexd.py [hunk 2]
```diff
@@ -78,7 +78,13 @@ def delete_record_via_api(
                 "Authorization": f"bearer {access_token}",
             }
         else:
-            headers = pytest.auth_headers[user]
+            auth = Gen3Auth(refresh_token=pytest.api_keys[user], endpoint=self.BASE_URL)
```
Comment (haraprasadj) [inline gen3-integration-tests/services/indexd.py:81]: can this also be restructured like in line 56 above?

Comment (krishnaa05) [reply gen3-integration-tests/services/indexd.py:76]: Done

**Pull request merged**